client/state/jetpack-connection-health/selectors/is-requesting-jetpack-connection-health-status.js

```diff
@@ -0,0 +1,16 @@
+import 'calypso/state/jetpack-connection-health/init';
+
+/**
+ * Returns true if we are currently making a request to retrieve the connection status. False otherwise.
+ * Returns null if the site is unknown, or there is no information yet.
+ * @param  {Object}   state    Global state tree
+ * @param  {number}   siteId   The ID of the site we're querying
+ * @returns {?boolean}          Whether the connection status is being requested
+ */
+export default function isRequestingJetpackConnectionHealthStatus( state, siteId ) {
```

Comment:
Seems like as I mentioned above we'll be able to remove this selector and its corresponding state subtree altogether.

Reply:
This is used at `JetpackConnectionHealthBanner` component.
Thanks for your comment!

Reply:
I saw the usage, but we could still consider removing it. Often from the perspective of the consumer component, whether the data is being requested is similar to whether the data is missing - the component doesn't care and treats both cases the same way.